ajax/libs/timbre/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "timbre",
+  "version": "14.11.25",
+  "filename": "timbre.js",
+  "description": "JavaScript library for objective sound programming",
+  "author": "nao yonamine <mohayonao@gmail.com>",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mohayonao/timbre.js.git"
+  },
+  "license": "MIT",
+  "homepage": "https://mohayonao.github.io/timbre.js",
+  "keywords": [
+    "timbre.js",
+    "timbre",
+    "audio",
+    "synth"
+  ],
+  "npmName": "timbre",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "timbre.dev.js",
+        "timbre.js",
+        "timbre.js.map"
```

Comment:
Maybe single line pattern `timbre.!(node*)*.+(js|map|css)`?

Reply:
The reason I use a complete file name is the author hasn't put `timbre.js` and `timbre.js.map` to the repo. So I think it's better we fetch the file we want certainly.

Reply:
I didn't see a need to use multi pattern here ... using the flexible method is preferred.